spec/controllers/exchanges/employer_applications_controller_spec.rb

```diff
@@ -118,11 +120,87 @@
     before :each do
       allow(hbx_staff_role).to receive(:permission).and_return(double('Permission', can_modify_plan_year: true))
       sign_in(user)
-      get :get_term_reasons, { reason_type_id: "term_actions_nonpayment" },  format: :js
+      get :term_reasons, { reason_type_id: "term_actions_nonpayment" },  format: :js
```

Comment:
is this required?

Reply:
It's because of rubocop issue I have updated the method name.